packages/amplify_auth_plugin_interface/lib/src/Password/PasswordOptions.dart

```diff
@@ -13,10 +13,23 @@
  * permissions and limitations under the License.
  */
 
-abstract class PasswordOptions {
+import 'ConfirmResetPasswordOptions.dart';
+import 'ResetPasswordOptions.dart';
+import 'UpdatePasswordOptions.dart';
+
+/// Previously used as the shared password options for [updatePassword],
```

Comment:
*Note:* PasswordOptions was being used for three separate requests. It is being replaced with three specific options classes. Having PasswordOptions mixin the new classes will allow for backwards compatibility.